core/jsoncollector.class.inc.php

```diff
@@ -327,6 +320,15 @@ public function Fetch()
                     throw new Exception("Missing columns in the Json file.");
                 }
             }
+            //check if all expected fields are in array. If not add it with null value
+            foreach($this->aCSVHeaders as $sHeader)
+            {
+               if(!isset($aDataToSynchronize[$sHeader]))
+               {
+                   $aDataToSynchronize[$sHeader]="";
```

Comment:
This is not a null value:
```suggestion
                   $aDataToSynchronize[$sHeader]=null;
```